markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -11,414 +11,298 @@ If you create a new module with the goal of contributing the code to *nf-core*,
+If the module doesn't exist on `nf-core/modules`:
 
-```bash
-fgbio --tmp-dir=tmpFolder \\
-    FastqToBam \\
-    -i read1_fastq.gz read2_fastq.gz \\
-    -o "sampleID_umi_converted.bam" \\
-    --read-structures "+T 12M11S+T" \\
-    --sample "sampleID" \\
-    --library "sampleID"
-```
+- Please create a [new issue](https://github.com/nf-core/modules/issues/new?assignees=&labels=new%20module&template=new_nodule.md&title=new%20module:) before adding it
+- Set an appropriate subject for the issue e.g. `new module: fastqc`
+- Add yourself to the `Assignees` so we can track who is working on the module
 
-Here you should first identify:
-
-- the inputs you need, which are mandatory
-- the inputs / arguments, which are optional
-- the outputs
-- any value or variable you might need, associated with the sample (for example, the sample ID or other metadata)
-
-### Inputs and Outputs
-
-As described in the guidelines, any sample-specific information should be passed as an input, as part of a groovy map called *meta*. This is part of a tuple which includes the read file(s).
-
-In our case, FGBIO also has a mandatory argument, which is not sample-specific, i.e. the read structure: this refers to the position and structure of the UMI barcode in the read. Such information will be the same for all samples and characteristics of the kit used to prepare the sequencing library. Since it is not sample specific, we will not include it in the *meta* map. Since it is a mandatory argument, we have decided to add it to the input list: in this way, it will be visible to others who wish to reuse this module, and it will be described explicitly in the metadata YAML file.
-
-Therefore, once we modify the template accordingly, our inputs and outputs will look like this:
-
-```nextflow
-input:
-    tuple val(meta), path(reads)
-    val(read_structure)
-
-output:
-    tuple val(meta), path("*_umi_converted.bam"), emit: umibam
-    path "*.version.txt"          , emit: version
-```
-
-### Passing options.args
-
-Any optional parameter should be passed within a groovy map called options, and identified as a string within *args*.
-In our specific case, we are not using any optional argument and therefore we will include the module and initialise this with an empty string in our test workflow.
-
-```nextflow
-include { FGBIO_FASTQTOBAM } from '../../../../modules/fgbio/fastqtobam/main.nf' addParams( options: [args: ''] )
-```
-
-But we will add this into the script code, so any other user who might want to execute the function with additional arguments will be able to.
-The code of the script will therefore look like this, once we have substituted inputs and outputs as appropriate.
-
-```bash
-mkdir tmpFolder
-
-fgbio --tmp-dir=${PWD}/tmpFolder \\
-FastqToBam \\
--i ${reads} \\
--o "${prefix}_umi_converted.bam" \\
---read-structures $read_structure \\
---sample ${meta.id} \\
---library ${meta.id} ${options.args}
-```
```

Comment:
Again, I understand the goal is to make the page lean but this tutorial basically links to existing code of the FASTQC modules: a novice should follow the links and read the code of the entire script, making a connection between the instructions on this page and a ready-to-use finalised code.
Personally, I believe explaining the code bit by bit is part of the goal of a tutorial, although I understand it takes a bit of effort to write (and maintain).